app/controllers/shelters_controller.rb

```diff
@@ -58,4 +58,16 @@ def shelter_params
                   :zip)
   end
 
+  def empty_params
+    empty_params = []
+    empty_params << "Name" if params[:name] == ""
+    empty_params << "Address" if params[:address] == ""
+    empty_params << "City" if params[:city] == ""
+    empty_params << "State" if params[:state] == ""
+    empty_params << "Zip" if params[:zip] == ""
+    empty_params.join(", ")
+  end
```

Comment:
if any of the validated fields are not entered, add to a list to display in a flash message